### PR TITLE
Add aos-addons

### PR DESCRIPTION
**Pull request by iartemenko**
Install nesesery for aos_servicemanager data and service files.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aos_addons/files/fcrypt.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "CACert": "var/aos/fcrypt/rootCA.crt.pem",
```
Comment (al1img) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aos_addons/files/fcrypt.json]: Use absolute path: /var/aos/fcrypt...

**Comment by andr2000** [comment]
please spell check the commit message

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aos_addons/aos-addons.bb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+SUMMARY = "Data and service files for aos_servicemanager"
+DESCRIPTION = "Data and service files for aos_servicemanager which will be using for tests"
```
Comment (andr2000) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/aos_addons/aos-addons.bb]: be using -> be used

**Comment by andr2000** [comment]
Is there a chance this recipe can be re-used? I mean that we might want to have .bb in meta-xt-images and .bbappend in the product

**Comment by aanisov** [comment]
Updated to #60

**Pull request closed without merging**